src/Tizen.Multimedia/AudioManager/AudioDucking.cs

```diff
@@ -77,11 +77,28 @@ public bool IsDucked
         /// <summary>
         /// Activate audio ducking
         /// </summary>
-        /// <param name="duration">The duration for ducking.</param>
-        /// <param name="ratio">The volume ratio after ducked.</param>
+        /// <param name="duration">The duration(milisecond) for ducking. Valid range is 0 to 3000, inclusive.</param>
+        /// <param name="ratio">The volume ratio after ducked. Valid range is 0.0 to 1.0, exclusive.</param>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///     <paramref name="duration"/> is less than 0 or greater than 3000.<br/>
+        ///     -or-<br/>
+        ///     <paramref name="ratio"/> is less then 0.0 or greater than 1.0.<br/>
```

Comment:
_> is less then 0.0 or greater than 1.0._
The range of this parameter is 0.0 < ratio < 1.0. If so, the exception condition is supposed to be  "less 'equal' th'a'n 0.0 or greater equal than 1.0.".

If you happen to agree with the previous my comment - allowable 0.0 -, you may only need to fix the "less th'e'n" and "greater 'equal' than 1.0."

Reply:
"less than or equal to", "greater than or equal to" is more accurate phrase.

Reply:
Done.